.coveralls.yml

```diff
@@ -0,0 +1,3 @@
+src_dir: .
```

Comment:
Coverage reporting on coveralls.io seems to works just fine with this. I don't really mind adding this file but would prefer if we don't add extra meta data files if not really needed.